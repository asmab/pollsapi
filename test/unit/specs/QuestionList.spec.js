import Vue from 'vue'
import { shallowMount } from '@vue/test-utils'
import QuestionList from '@/components/QuestionsList '

describe('Shipping component', () => {
    let wrapper

    beforeEach(() => {
        wrapper = shallowMount(QuestionList)
    })

    it('QuestionList component looks as expected', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })
})