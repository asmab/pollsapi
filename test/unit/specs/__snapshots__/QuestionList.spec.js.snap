// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shipping component QuestionList component looks as expected 1`] = `
<div>
    <h1>Fetching data .....</h1>
</div>
`;
